.agents/skills/react-best-practices/SKILL.md

```
@@ -0,0 +1,42 @@
+name: react-best-practices
```

markdownlint-cli2 (0.20.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)